Backend/controllers/adminc.js

```diff
@@ -0,0 +1,16 @@
+let admin = [];
+const createAdmin = async (req, res) => {
+  const { email, password } = req.body;
+  admin.push(email);
+  //   console.log(email);
+  res.status(201).json({ message: "Admin created successfully", email });
+};
+const getAdminById = async (req, res) => {
+  const id = req.params.adminId;
+  console.log(admin[0]);
+  res.json({ admin: admin[0] });
+};
+const updateAdmin = async (req, res) => {};
```

Comment:
In case the method is not implemented, provide a comment that signifies it is under progress. And have it return an error 500, stating that it has not been implemented yet.

Reply:
I leave error handing as of now i just write dummy function . I implement the proper function with try catch blocks as soon as I get all necessary data. during connecting to frontend.